courses/models.py

```diff
@@ -152,6 +162,16 @@ def duration(self):
         """Gets the duration from the associated Page if it exists"""
         return self.page.duration if self.page else None
 
+    @property
+    def time_commitment(self):
+        """Gets the time commitment from the associated Page if it exists"""
+        return self.page.time_commitment if self.page else None
+
+    @property
+    def format(self):
+        """Gets the format from the associated Page if it exists"""
+        return self.page.format if self.page else None
+
```

Comment:
can we move `format` and `time_commitment` definitions to utils to reuse.

Reply:
other definitions are placed in the  models as property. May be not.